### ISSUE TITLE
Query failed on launch

### ISSUE THREAD

**Issue by neissa** [rapidloop/pgmetrics#9]
./pgmetrics --no-password
pgmetrics: pg_roles/pg_auth_members query failed: pq: bigint en dehors des limites

psql -V
psql (PostgreSQL) 10.3 (Debian 10.3-1.pgdg80+1)

**Comment by mdevan** [comment]
Does any of your roles have "valid until" set as Infinity?

```
select rolname, rolvaliduntil from pg_roles where rolvaliduntil = 'Infinity';
```

If so, this should be fixed in pgmetrics v1.1. Please download the [v1.1 from here](https://github.com/rapidloop/pgmetrics/releases/tag/v1.1.0) and try with that.

**Comment by mdevan** [comment]
Updated documentation, thanks.
